admin-manual.md

```diff
@@ -0,0 +1,187 @@
+|---------------------------|------------------------------------------------------------|
+| `issuetracker.properties` | settings for an issue tracking system, such as JIRA or RT. |
+| `mail.properties`         | email settings so that MISO can send emails to users.      |
+| `security.properties`     | properties to set the security environment (see below).    |
+| `submission.properties`   | properties to set the submission environment (see below).  |
+
+## Security Environment
+MISO can use either LDAP or SQL as an authentication mechanism. The mechanism
+is set in both `security.properties` and the `setenv.sh` file and both must be
+the same.
+
+If you are using the database (aka JDBC), set the security method to `jdbc`. The default configuration should work properly.
+
+For using LDAP, set the security method to `ldap`. Additional settings are
+needed for LDAP in the `security.properties`. Talk to your LDAP administrator.
+
+# Setting Up the Notification Server
+The notification server is a Java daemon that scans the paths containing
+sequencer output. It is not required for a functioning MISO install, but
+without it, sequencer runs must be added manually. Configuration for
+`systemd`-based Linux systems is provided.
+
+Create the default configuration:
+
+    mkdir /srv/notification-server
+    cp notification-server/service/notification.properties /srv/notification-server
+    cp notification-server/service/miso-notification.service /etc/systemd/system
+
+Edit `notification.properties`:
+
+1. Uncomment necessary `<service>.dataPaths` line and add comma-separated paths to data folders to scan
+1. Replace `localhost:8080` with URL to MISO web server
+
+After building and deploying the JAR:
+
+   sudo systemctl daemon-reload
+   sudo systemctl enable miso-notification.service
+   sudo systemctl start miso-notification.service
+
+The service should start up. You can inspect `stdout` in `/srv/notification-server/notification/notification.log` file, and `stderr` by `sudo journalctl -f -u miso-notification`.
+
+# Building the Applicaton
+Building the application is done by:
+
+    mvn clean package -P external
+
+There will be two important build artifacts:
+
+- `miso-web/target/ROOT.war`
+- `notification-server/target/notification-server-*.one-jar.jar`
+
+# Releasing and Upgrading
+
+To install or upgrade, perform the following steps:
+
+1. Backup your existing database.
+1. Stop Tomcat.
+1. Migrate the database to the newest version. (Described below.)
+1. Copy the `ROOT.war` from the build to `$TOMCAT_ROOT/webapps`.
+1. Remove `$TOMCAT_ROOT/webapps/ROOT`.
+1. Start Tomcat.
+1. Stop the notification server.
+1. Copy the `notification-server-*.one-jar.jar` to `/srv/notification-server/notification-server.jar`.
+1. Restart the notification server.
+
+## Migrating the database
+Updating the database (or setting it up initially) will apply patches to the database using Flyway using the `ROOT.war`.
+
+    cd ${FLYWAY}
```

Comment:
Is the ROOT.war in the Flyway root? Last I heard it was in the $CATALINA_HOME/webapps directory.